x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/stringcontains/StringContainsUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.eql.expression.function.scalar.stringcontains;
+
+import java.util.Locale;
+
+import static org.elasticsearch.common.Strings.hasLength;
+
+final class StringContainsUtils {
```

Comment:
Why this class? Shouldn't this method belong to a class that has other String utility method like this one here? And is this really needed to be placed in a separate class?

Reply:
I think these methods could be in function.scalar.string.StringUtils

Reply:
Moved the method to `function.scalar.string.StringUtils`.